tests/Features/Properties/ModelPropertyExtension.php

```diff
@@ -95,4 +96,16 @@ public function testWriteIdPropertyWhenMigrationsCouldntBeRead(): bool
 
         return $group->save();
     }
+
+    /** @test */
+    public function it_knows_name_of_primary_key_column(): void
+    {
+        $model = new CustomPrimaryKeyColumn();
+        $primaryKey = $model->custom_primary_key_name;
```

Comment:
Can you return this here and check the return type is correctly recognized as an integer?